<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像处理</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_start">
    <property name="title">
     <string>开始(&amp;S)</string>
    </property>
    <widget class="QMenu" name="menu_recent">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>最近打开的文件</string>
     </property>
     <addaction name="actionClear"/>
    </widget>
    <addaction name="actionOpenImage"/>
    <addaction name="menu_recent"/>
    <addaction name="actionOpenAsLayer"/>
    <addaction name="actionCloseImage"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_transform">
    <property name="title">
     <string>变换(&amp;T)</string>
    </property>
    <addaction name="actionFlipHorizontal"/>
    <addaction name="actionFlipVertical"/>
    <addaction name="separator"/>
    <addaction name="actionClockwiseRotate90"/>
    <addaction name="actionAntiClockwiseRotate90"/>
    <addaction name="actionRotate180"/>
    <addaction name="actionRotateZoom"/>
    <addaction name="actionPerspective"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_color">
    <property name="title">
     <string>颜色(&amp;C)</string>
    </property>
    <widget class="QMenu" name="menu_color_auto">
     <property name="title">
      <string>自动</string>
     </property>
     <addaction name="actionEqualize"/>
     <addaction name="actionWhiteBalance"/>
     <addaction name="actionColorEnhance"/>
    </widget>
    <addaction name="actionColorBalance"/>
    <addaction name="actionHSL"/>
    <addaction name="actionColorize"/>
    <addaction name="actionBrightAndContrasty"/>
    <addaction name="actionThreshold"/>
    <addaction name="actionLevels"/>
    <addaction name="actionCurve"/>
    <addaction name="actionPosterize"/>
    <addaction name="separator"/>
    <addaction name="actionGrayScale"/>
    <addaction name="actionInverting"/>
    <addaction name="menu_color_auto"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_filter">
    <property name="title">
     <string>滤镜(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menu_blur">
     <property name="title">
      <string>模糊</string>
     </property>
     <addaction name="actionZoomBlur"/>
     <addaction name="actionRadialBlur"/>
     <addaction name="actionGaussianBlur"/>
     <addaction name="actionMotionBlur"/>
     <addaction name="actionSurfaceBlur"/>
     <addaction name="actionMosaic"/>
    </widget>
    <widget class="QMenu" name="menu_edge">
     <property name="title">
      <string>边缘检测</string>
     </property>
     <addaction name="actionSobelEdge"/>
     <addaction name="actionLaplacianEdge"/>
     <addaction name="actionCanny"/>
    </widget>
    <widget class="QMenu" name="menu_art">
     <property name="title">
      <string>艺术</string>
     </property>
     <addaction name="actionCanvas"/>
     <addaction name="actionEmboss"/>
    </widget>
    <widget class="QMenu" name="menu_enhance">
     <property name="title">
      <string>增强</string>
     </property>
     <addaction name="actionRedEyeRemoval"/>
     <addaction name="actionSharpen"/>
     <addaction name="actionUnsharpMask"/>
    </widget>
    <widget class="QMenu" name="menu_light">
     <property name="title">
      <string>光照</string>
     </property>
     <addaction name="actionSoftGlow"/>
    </widget>
    <addaction name="menu_blur"/>
    <addaction name="menu_edge"/>
    <addaction name="menu_art"/>
    <addaction name="menu_enhance"/>
    <addaction name="menu_light"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <widget class="QMenu" name="menu_tools_selector">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>选择工具(&amp;S)</string>
     </property>
     <property name="toolTipsVisible">
      <bool>false</bool>
     </property>
     <addaction name="actionRectangle"/>
     <addaction name="actionEllipse"/>
     <addaction name="actionLasso"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menu_tools_selector"/>
    <addaction name="actionHistogram"/>
    <addaction name="actionBrowse"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
   </widget>
   <addaction name="menu_start"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_transform"/>
   <addaction name="menu_color"/>
   <addaction name="menu_filter"/>
   <addaction name="menu_tools"/>
   <addaction name="menu_view"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenImage"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
  </widget>
  <widget class="RichStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpenImage">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/start/Image/stock-start-open.png</normaloff>:/icon/start/Image/stock-start-open.png</iconset>
   </property>
   <property name="text">
    <string>打开图像(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开图像(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseImage">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/start/Image/stock-start-close.png</normaloff>:/icon/start/Image/stock-start-close.png</iconset>
   </property>
   <property name="text">
    <string>关闭图像(&amp;W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/start/Image/stock-start-exit.png</normaloff>:/icon/start/Image/stock-start-exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/start/Image/stock-start-save.png</normaloff>:/icon/start/Image/stock-start-save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFlipHorizontal">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/transform/Image/stock-trans-flip-horizontal.png</normaloff>:/icon/transform/Image/stock-trans-flip-horizontal.png</iconset>
   </property>
   <property name="text">
    <string>水平翻转(&amp;H)</string>
   </property>
  </action>
  <action name="actionFlipVertical">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/transform/Image/stock-trans-flip-vertical.png</normaloff>:/icon/transform/Image/stock-trans-flip-vertical.png</iconset>
   </property>
   <property name="text">
    <string>垂直翻转(&amp;V)</string>
   </property>
  </action>
  <action name="actionClockwiseRotate90">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/transform/Image/stock-trans-rotate-90.png</normaloff>:/icon/transform/Image/stock-trans-rotate-90.png</iconset>
   </property>
   <property name="text">
    <string>顺时针旋转 90 度</string>
   </property>
  </action>
  <action name="actionAntiClockwiseRotate90">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/transform/Image/stock-trans-rotate-270.png</normaloff>:/icon/transform/Image/stock-trans-rotate-270.png</iconset>
   </property>
   <property name="text">
    <string>逆时针旋转 90 度</string>
   </property>
  </action>
  <action name="actionRotate180">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/transform/Image/stock-trans-rotate-180.png</normaloff>:/icon/transform/Image/stock-trans-rotate-180.png</iconset>
   </property>
   <property name="text">
    <string>旋转 180 度</string>
   </property>
  </action>
  <action name="actionGrayScale">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/color/Image/stock-color-desaturate.png</normaloff>:/icon/color/Image/stock-color-desaturate.png</iconset>
   </property>
   <property name="text">
    <string>去色</string>
   </property>
  </action>
  <action name="actionInverting">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/color/Image/stock-color-invert.png</normaloff>:/icon/color/Image/stock-color-invert.png</iconset>
   </property>
   <property name="text">
    <string>反相</string>
   </property>
  </action>
  <action name="actionThreshold">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/color/Image/stock-color-threshold.png</normaloff>:/icon/color/Image/stock-color-threshold.png</iconset>
   </property>
   <property name="text">
    <string>阈值</string>
   </property>
  </action>
  <action name="actionColorBalance">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/color/Image/stock-color-color-balance.png</normaloff>:/icon/color/Image/stock-color-color-balance.png</iconset>
   </property>
   <property name="text">
    <string>色彩平衡</string>
   </property>
  </action>
  <action name="actionHSL">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/color/Image/stock-color-hue-saturation.png</normaloff>:/icon/color/Image/stock-color-hue-saturation.png</iconset>
   </property>
   <property name="text">
    <string>色调-亮度-饱和度</string>
   </property>
  </action>
  <action name="actionBrightAndContrasty">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/color/Image/stock-color-brightness-contrast.png</normaloff>:/icon/color/Image/stock-color-brightness-contrast.png</iconset>
   </property>
   <property name="text">
    <string>亮度-对比度</string>
   </property>
  </action>
  <action name="actionPosterize">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/color/Image/stock-color-posterize.png</normaloff>:/icon/color/Image/stock-color-posterize.png</iconset>
   </property>
   <property name="text">
    <string>色调分离</string>
   </property>
  </action>
  <action name="actionEqualize">
   <property name="text">
    <string>色调均化</string>
   </property>
   <property name="iconText">
    <string>色调均化</string>
   </property>
  </action>
  <action name="actionWhiteBalance">
   <property name="text">
    <string>白平衡</string>
   </property>
  </action>
  <action name="actionColorEnhance">
   <property name="text">
    <string>颜色增强</string>
   </property>
  </action>
  <action name="actionPerspective">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/transform/Image/stock-trans-perspective.png</normaloff>:/icon/transform/Image/stock-trans-perspective.png</iconset>
   </property>
   <property name="text">
    <string>透视变换</string>
   </property>
  </action>
  <action name="actionColorize">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/color/Image/stock-color-colorize.png</normaloff>:/icon/color/Image/stock-color-colorize.png</iconset>
   </property>
   <property name="text">
    <string>单色化</string>
   </property>
  </action>
  <action name="actionLevels">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/color/Image/stock-color-levels.png</normaloff>:/icon/color/Image/stock-color-levels.png</iconset>
   </property>
   <property name="text">
    <string>色阶</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/Image/stock-histogram.png</normaloff>:/icon/Image/stock-histogram.png</iconset>
   </property>
   <property name="text">
    <string>直方图</string>
   </property>
  </action>
  <action name="actionCurve">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/color/Image/stock-color-curves.png</normaloff>:/icon/color/Image/stock-color-curves.png</iconset>
   </property>
   <property name="text">
    <string>曲线</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/start/Image/stock-start-clear.png</normaloff>:/icon/start/Image/stock-start-clear.png</iconset>
   </property>
   <property name="text">
    <string>（无）</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/start/Image/stock-start-save-as.png</normaloff>:/icon/start/Image/stock-start-save-as.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRotateZoom">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/transform/Image/stock-trans-rotate-zoom.png</normaloff>:/icon/transform/Image/stock-trans-rotate-zoom.png</iconset>
   </property>
   <property name="text">
    <string>旋转/缩放</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/tools/Image/stock-tool-rect-select.png</normaloff>:/icon/tools/Image/stock-tool-rect-select.png</iconset>
   </property>
   <property name="text">
    <string>矩形选择(&amp;R)</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/tools/Image/stock-tool-ellipse-select.png</normaloff>:/icon/tools/Image/stock-tool-ellipse-select.png</iconset>
   </property>
   <property name="text">
    <string>椭圆选择(&amp;E)</string>
   </property>
  </action>
  <action name="actionLasso">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/tools/Image/stock-tool-free-select.png</normaloff>:/icon/tools/Image/stock-tool-free-select.png</iconset>
   </property>
   <property name="text">
    <string>套索选择(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>套索选择(F)</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/icon/tools/Image/stock-tool-browse.png</normaloff>:/icon/tools/Image/stock-tool-browse.png</iconset>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </action>
  <action name="actionZoomBlur">
   <property name="text">
    <string>径向模糊</string>
   </property>
  </action>
  <action name="actionGaussianBlur">
   <property name="text">
    <string>高斯模糊</string>
   </property>
  </action>
  <action name="actionSobelEdge">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionLaplacianEdge">
   <property name="text">
    <string>拉普拉斯</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionMotionBlur">
   <property name="text">
    <string>运动模糊</string>
   </property>
  </action>
  <action name="actionRadialBlur">
   <property name="text">
    <string>旋转模糊</string>
   </property>
  </action>
  <action name="actionMosaic">
   <property name="text">
    <string>马赛克</string>
   </property>
  </action>
  <action name="actionSurfaceBlur">
   <property name="text">
    <string>表面模糊</string>
   </property>
  </action>
  <action name="actionCanvas">
   <property name="text">
    <string>画布</string>
   </property>
  </action>
  <action name="actionRedEyeRemoval">
   <property name="text">
    <string>去除红眼</string>
   </property>
  </action>
  <action name="actionSharpen">
   <property name="text">
    <string>锐化</string>
   </property>
  </action>
  <action name="actionUnsharpMask">
   <property name="text">
    <string>虚光蒙版</string>
   </property>
  </action>
  <action name="actionEmboss">
   <property name="text">
    <string>浮雕</string>
   </property>
  </action>
  <action name="actionSoftGlow">
   <property name="text">
    <string>柔光</string>
   </property>
  </action>
  <action name="actionOpenAsLayer">
   <property name="text">
    <string>作为图层打开</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RichStatusBar</class>
   <extends>QStatusBar</extends>
   <header>statusbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
